resistance/resistance.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("The program calculates the resistance R of two resistors.")
+	fmt.Println("Enter the resistance of two resistors:")
+	var r1, r2 float64
+	fmt.Scan(&r1, &r2)
+	Rs := r1 + r2
+	Rp := 1/r1 + 1/r2
```

Comment:
Please note that it's `1/rp = 1/r1+1/r2`, which means that `rp` is smth different!